### DISCUSSION TITLE
how to move the HOME \ BACK minitoolbar ?

### DISCUSSION THREAD

**Discussion by terranauts** [AndreyPavlenko/Fermata#565] [Q&A]
hi, I am trying to figure out how to move that mini toolbar which has the home and go back functions, as this toolbar gets in the way of various things such as website browsing. thanks for letting me know how to move the mini toolbar around the screen and out of the way.

**Comment by AndreyPavlenko** [comment]
Do you mean the floating button? You can move it.

**Reply by terranauts** [reply]
thanks for the quick response. 
I have tried moving it by long pressing either icon including in between the icons, as well as double pressing the same way, but to no avail. this is on my pixel 9 pro XL phone. previously on my pixel 6 pro I thought I could move it by long pressing on the icons and then dragging, but that doesn't work now with my newer phone, though all else is working correctly. what was the way to move that many toolbar out of the way exactly and I will try that?

**Reply by AndreyPavlenko** [reply]
To move the floating (rounded) button, just touch and swipe.
